taxi/forms.py

```diff
@@ -0,0 +1,57 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+from django.contrib.auth.forms import UserCreationForm
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name",
+            "last_name",
+            "license_number",
+        )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    DRIVER_LICENSE_LENGTH = 8
+
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != DriverLicenseUpdateForm.\
+                DRIVER_LICENSE_LENGTH:
```

Comment:
it seems to me, it is better just to write an 8 here, to avoid line breaking, or use much smaller name for constant